lib/rmt/cli/base.rb

```diff
@@ -16,6 +16,10 @@ def help(shell, subcommand = false)
       class_options_help(shell)
 
       shell.say "Run '#{basename} help [COMMAND]' for more information on a command and its subcommands."
+
+      shell.say
+      shell.say 'Do you have suggestions for improvement? Let us know!'
```

Comment:
Would love to get the opinion from @jezuhke and @thutterer - here it is quite important which tone/emotion message has. Is it inviting enough to provide feedback?

Reply:
Or maybe something like: Let us know -> We would love to here from you